content/docs/command-reference/remote/modify.md

```diff
@@ -250,27 +250,71 @@ For more information on configuring Azure Storage connection strings, visit
 
 Please check out
 [Setup a Google Drive DVC Remote](/doc/user-guide/setup-google-drive-remote) for
-a full guide on configuring Google Drives for use as DVC remote storage,
-including obtaining the necessary credentials, and how to form `gdrive://` URLs.
+a full guide on using Google Drive as DVC remote storage, including how to form
+`gdrive://` URLs, and high-performance/enterprise setups.
 
 - `url` - remote location URL.
 
   ```dvc
   $ dvc remote modify myremote url gdrive://root/path/to/folder
   ```
 
-- `gdrive_client_id` - Google Project's OAuth 2.0 **client ID**.
+- `gdrive_client_id` - Google Project's **client ID** for authentication with
+  OAuth 2.0.
 
   ```dvc
   $ dvc remote modify myremote gdrive_client_id <client ID>
   ```
 
-- `gdrive_client_secret` - Google Project's OAuth 2.0 **client secret**.
+- `gdrive_client_secret` - Google Project's **client secret** for authentication
+  with OAuth 2.0.
 
   ```dvc
   $ dvc remote modify myremote gdrive_client_secret <client secret>
   ```
 
+- `gdrive_use_service_account` - instructs DVC to authenticate using a
```

Comment:
check Google Storage add/modify with some structure/suggestions on these service account options

Reply:
Not sure I understand this comment. But remote add doesn't even mention this directly, just links to the full guide. I did already add a link to the service acct section of the full guide under this line. I also added info about which params require which other params. I feel like that may be enough? Otherwise may have to rewrite these cmd ref sections significantly (maybe open a separate issue?)

Reply:
Sure, up to you.

I meant something like this:

Adding structure like this? Just to combine them and include a link to the full guide and service account google docs? 

<img width="676" alt="Screen Shot 2020-04-01 at 6 41 54 PM" src="https://user-images.githubusercontent.com/3659196/78202237-9c7b1200-7448-11ea-95ff-4fd1c39ff03e.png">

Reply:
Again, up to you - looks good as it is now.

Reply:
You may be right about having a separate section but I have doubts about how GCS remotes work to make sure the section is useful in both types. Leaving this as is for now. Sent you some Qs on slack

Reply:
OK I added a little heading and paragraph about service accounts to link to the guide and separate "regular" params from service acct params.